Default.sublime-commands

```diff
@@ -162,4 +162,9 @@
         "caption": "LSP: Toggle Inlay Hints",
         "command": "lsp_toggle_inlay_hints",
     },
+    // {
+    //     "caption": "LSP: Fold All Block Comments",
```

Comment:
I wonder if it's always true that it's only gonna apply to block comments. I don't think spec mentions "block" comments specifically.

Reply:
True, I just copied the label from VS Code here. But I had a better idea, instead of "block comments" we can use "comment blocks", this should include both block comments and consecutive line comments, so it should be accurate (assuming that no server gets the idea to add inline folding for single-line comments, but that would be stupid and I think we can also rule that out due to VS Code having no support for inline folding at the moment).